eth/api.go

```diff
@@ -325,6 +325,42 @@ func (api *PublicDebugAPI) DumpBlock(blockNr rpc.BlockNumber, typ *string) (stat
 	return publicState.RawDump(false, false, true), nil
 }
 
+// Quorum
+// DumpAddress retrieves the state of an address at a given block.
+// Quorum adds an additional parameter to support private state dump
+func (api *PublicDebugAPI) DumpAddress(address common.Address, blockNr rpc.BlockNumber) (state.DumpAccount, error) {
```

Comment:
There's an opportunity to extract the below duplicated code which does retrieval of states and reuse in both `DumpBlock` and `DumpAddress`.
`DumpBlock` also has Quorum modification footprints so it makes sense to do the refactoring.

Reply:
Done, thanks!